### DISCUSSION TITLE
What's the command to hide source image when running detect.py

### DISCUSSION THREAD

**Discussion by xoxolo** [ultralytics/yolov5#7347] [Q&A] [answered]
Good day Guys,
I'm just looking for the labels, I don't need to see images "live" of my source once bounding boxed.
Can't find the command to hide them during the process.
Thanks for you help!
Have a good day!
O

**Comment by glenn-jocher** [comment, marked as answer]
@xoxolo `view_img` controls showing images during inference:
https://github.com/ultralytics/yolov5/blob/302b00b5f4b93bb6cdb3c651dc9f06b66d06016d/detect.py#L59

**Reply by xoxolo** [reply]
Thanks a lot Glenn

**Reply by xoxolo** [reply]
> @xoxolo `view_img` controls showing images during inference:
> 
> https://github.com/ultralytics/yolov5/blob/302b00b5f4b93bb6cdb3c651dc9f06b66d06016d/detect.py#L59

Still showing while running the script.
I run it on a youtube feed.
- did try on static images, same thing.
- Any clue?

**Reply by glenn-jocher** [reply]
Modify it wherever you need to in detect.py.

**Reply by xoxolo** [reply]
sorry was not clear enough in my reply.
 view_img=False,  # show results  or view_img=True,  # show results 
...are giving me the same results. Results are displayed.

**Reply by glenn-jocher** [reply]
@xoxolo yes I know, this variable is updated dynamically throughout detect.py based on source type. Modify it as you see fit where you need it in detect.py.

**Reply by xoxolo** [reply]
Hi Glenn,
I've been searching ways to do, I did forgot to mention, I'm running detect.py on a live youtube feed.
Can you guide me a bit?
My only goal, is to: hide the results (feed with bounding boxes) to appear while I'm running the script.
Much appreciated.
my code look like: Detect.py --source https://www.youtube.com/watch?v=1EiC9bvVGnk --classes 0 2 5 7 --save-txt

**Reply by glenn-jocher** [reply]
@xoxolo you can change `view_img=False`:
https://github.com/ultralytics/yolov5/blob/c4cb7c684ad6baf4081eb59ee1c27aa97bed4ebe/detect.py#L96-L98
